test/test_librosa_compatibility.py

```diff
@@ -46,10 +54,6 @@ def test_griffinlim(self):
         assert torch.allclose(ta_out, lr_out, atol=5e-5)
 
     def _test_create_fb(self, n_mels=40, sample_rate=22050, n_fft=2048, fmin=0.0, fmax=8000.0):
-        # Using a decorator here causes parametrize to fail on Python 2
-        if not IMPORT_LIBROSA:
-            raise unittest.SkipTest('Librosa is not available')
-
```

Comment:
This is not a change part of moving the tests :)

Reply:
Yeah, but now the librosa availability check is performed automatically for all the tests under `test_librosa_consistency::TestFunctional`, so it's redundant and since it does not affect the test logic itself, I went beyond moving and removed it.